relbot/github_chat_monitor.py

```diff
@@ -91,7 +91,7 @@ def is_valid_name(s: str):
 
 
 def parse_github_urls(data) -> List[GitHubIssue]:
-    matches = re.findall(r"(https://github.com/[^\s#]+)", data)
+    matches = re.findall(r"(https://github\.com/[-.\w]+/[-.\w]+/(?:pull|issues)/\d+)", data)
```

Comment:
I escaped the dot because previously it matched any character.

I think that the code below would become a lot simpler if we got only the `\d+` part and used it directly. What do you think?